edx_proctoring/migrations/0013_auto_20210111_1728.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.17 on 2021-01-11 17:28
```

Comment:
can you add a descriptive filename to this migration?

Reply:
Yes, thank you for catching that!